integrationtest/integrationtest.go

```diff
@@ -67,20 +68,44 @@ type Helper struct {
 	protocol                      atomic.Value
 	t                             *testing.T
 	ConfigDir                     string
-	HTTPSProxyServerAddr          string
-	LampshadeProxyServerAddr      string
-	QUICIETFProxyServerAddr       string
-	WSSProxyServerAddr            string
-	ShadowsocksProxyServerAddr    string
+	BaseServerAddr                string
+	HTTPSProxyServerPort          int32
+	LampshadeProxyServerPort      int32
+	QUICIETFProxyServerPort       int32
+	WSSProxyServerPort            int32
+	ShadowsocksProxyServerPort    int32
+	ShadowsocksmuxProxyServerPort int32
+	TLSMasqProxyServerPort        int32
+	HTTPSSmuxProxyServerPort      int32
+	HTTPSPsmuxProxyServerPort     int32
+	HTTPServerPort                int32
+	HTTPSServerPort               int32
+
+	// Deprecated: Use BaseServerAddr and HTTPSProxyServerPort instead.
+	HTTPSProxyServerAddr string
```

Comment:
Since this is a package used (a) exclusively for tests and (b) only by our own projects, you don't need to deprecate these.  You can just make a breaking change to the API and synchronize that change when you update downstream projects like `lantern-client`.